hooks/pre-commit.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# This is the pre-commit hook that runs each time with `git commit`
+# and checks syntax of the playbooks
+# To install run
+#   ln -s ../../hooks/pre-commit.sh .git/hooks/pre-commit
+
+echo "Running pre-commit hook"
+
+STASH_NAME="pre-commit-$(date +%s)"
+git stash save -q --keep-index $STASH_NAME
+
+echo "Checking the playbook syntax"
+ansible-playbook -i hosts site.yml --syntax-check
```

Comment:
Does it actually need `hosts`? Maybe `-i none` will do?

Reply:
fixed in e38f34c 

It produces a lot of warnings, though. Looks like Ansible have no option to suppress these warnings